HomeTask-3-Youtube-API/load-page.js

```diff
@@ -0,0 +1,63 @@
+function init() {
+  var formDivId = document.createElement("div");
+  formDivId.setAttribute('id', 'search-form');
+  var label = document.createElement("div");
+  label.setAttribute('id', 'label');
+  label.innerHTML = "YouTube Query Search";
+  formDivId.appendChild(label);
+
+  var lineBreak = document.createElement("br");
+  formDivId.appendChild(lineBreak);
+
+  var inputType = document.createElement("input");
+  inputType.setAttribute("type", "text");
+  inputType.setAttribute("id", "incomingQuery");
+  inputType.setAttribute("name", "inputQuery");
+  inputType.setAttribute("placeholder", "Enter the keyword");
+  inputType.setAttribute("size", 50);
+  formDivId.appendChild(inputType);
+
+  var button = document.createElement("button");
+  button.setAttribute("id", "querySearch");
+  button.setAttribute('onclick', 'searchByKeyword();');
+  button.innerHTML = "Search";
+  formDivId.appendChild(button);
+
+  var nextButton = document.createElement("button");
+  nextButton.setAttribute("id", "buttonNext");
+  nextButton.innerHTML = "Next Page";
+  nextButton.setAttribute("onclick", "nextPage()");
+  formDivId.appendChild(nextButton);
+
+  var lineBreak = document.createElement("br");
+  formDivId.appendChild(lineBreak);
+
+  var prevButton = document.createElement("button");
+  prevButton.setAttribute("id", "buttonPrev");
+  prevButton.innerHTML = "Previous Page";
+  prevButton.setAttribute("onclick", "prevPage()");
+  formDivId.appendChild(prevButton);
+
```

Comment:
Use const instead of var.. change everywhere

Reply:
Changed it everywhere in the code.